src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    public void draw();

```

Comment:
The 'public' modifier is redundant for interface methods. Interface methods are implicitly public, so you can safely omit this modifier.